metpy/plots/tests/test_util.py

```diff
@@ -88,3 +89,65 @@ def test_add_logo_invalid_size():
     fig = plt.figure(figsize=(9, 9))
     with pytest.raises(ValueError):
         add_metpy_logo(fig, size='jumbo')
+
+
+@pytest.mark.mpl_image_compare(tolerance={'1.4': 0.004}.get(MPL_VERSION, 0.01),
```

Comment:
Probably should be (in order to pass): `tolerance={'1.4': 0.069}.get(MPL_VERSION, 0.01)`